scripts/utils.sh

```diff
@@ -4,13 +4,17 @@ function die() {
   endc='\033[0m'
 
   if [[ $# -ge 1 ]]; then
-    echo -e "\n${red}>>> $@ <<<${endc}\n"
+    if [[ -t 0 ]]; then
```

Comment:
Stdout is file descriptor number 1 on Unixes, you're checking for stdout's TTYness here (and not stdin's aka FD=0 TTYness, as in `extract_version_from_gradle_file_stdin`)